modules/sagemaker/sagemaker-templates-service-catalog/README.md

```diff
@@ -49,11 +49,23 @@ The template contains an example CI/CD pipeline to deploy the model endpoints to
 
 The template is based on basic multi-account template from [AWS Enterprise MLOps Framework](https://github.com/aws-samples/aws-enterprise-mlops-framework/blob/main/mlops-multi-account-cdk/mlops-sm-project-template/README.md#sagemaker-project-stack).
 
+As part for third party code repository (like GitHub) integration, SageMaker templates will be able to manage (create, delete) repositories. As an example, if sagemaker-templates-service-catalog module model_deploy is used then it would create code repository with directly into owner account provided into configuration. Repository will be named after SageMaker project name in AWS account `{sagemaker-project}-deploy`. For example, if SageMaker project name is `aiops-abalone-model` then GitHub repository would be created with name `aiops-abalone-model-deploy`.
+
+> [!IMPORTANT] 
+> It is important to note AWS CodeCommit is no longer available to new customers. Existing customers of AWS CodeCommit can continue to use the service as normal. 
+
+## Prerequesites:
```

Comment:
This is only required if `GitHub` is chosen as VCS option, correct? Please add a statement to clarify there are two available options, and in case GitHub is chosen, these are the prerequisites